doc/jsk_pcl_ros/nodes/color_histogram_matcher.md

```diff
@@ -0,0 +1,63 @@
+ColorHistogramMatcher
+====================
+
+What Is This?
+-------------
+
+.. image:: images/color_histogram_matcher.png
```

Comment:
This is syntax for [rst](http://docutils.sourceforge.net/rst.html), so the filename should end with `.rst`.
If you'd like to use markdown. This should be written as:

```
![](images/color_histogram_matcher.png)
```